homeassistant/components/heatzy/climate.py

```diff
@@ -0,0 +1,295 @@
+"""Climate sensors for Heatzy."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from heatzypy.exception import HeatzyException
+
+from homeassistant.components.climate import (
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_NONE,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import HeatzyDataUpdateCoordinator
+from .const import (
+    CFT_TEMP_H,
+    CFT_TEMP_L,
+    CONF_ALIAS,
+    CONF_ATTR,
+    CONF_ATTRS,
+    CONF_MODE,
+    CONF_MODEL,
+    CONF_ON_OFF,
+    CONF_PRODUCT_KEY,
+    CONF_VERSION,
+    CUR_TEMP_H,
+    CUR_TEMP_L,
+    DOMAIN,
+    ECO_TEMP_H,
+    ECO_TEMP_L,
+    ELEC_PRO_SOC,
+    GLOW,
+    PILOTEV1,
+    PILOTEV2,
+)
+
+MODE_LIST = [HVACMode.HEAT, HVACMode.OFF]
+PRESET_LIST = [PRESET_NONE, PRESET_COMFORT, PRESET_ECO, PRESET_AWAY]
```

Comment:
Do these need to be lists (do their order matter)? Could they be sets instead? Should they live in `const`?

Reply:
Indeed I don't have to make constants because I use them only once to value the _attr_hvac_modes and _attr_preset_modes attributes. I fixed this